src/libs/TransactionUtils.ts

```diff
@@ -0,0 +1,358 @@
+import Onyx, {OnyxCollection} from 'react-native-onyx';
+import {format, parseISO, isValid} from 'date-fns';
+import CONST from '../CONST';
+import ONYXKEYS from '../ONYXKEYS';
+import DateUtils from './DateUtils';
+import * as NumberUtils from './NumberUtils';
+import {RecentWaypoints, ReportAction, Transaction} from '../types/onyx';
```

Comment:
@VickyStash this seems to be a tied to work which I done [here](https://github.com/Expensify/App/pull/27138/files) this PR is just waiting to be merged. 
And there I am changing `RecentWaypoints` to `RecentWaypoint` to follow TS guides. 
Just FYI

Reply:
@fabioh8010 @kubabutkiewicz Should this be put on hold till that PR is merged then?

Reply:
Yes, let's hold this one until https://github.com/Expensify/App/pull/27138/files is merged (it should be done soon).